bucket/regeditor.json

```
@@ -28,7 +28,7 @@
     },
     "pre_install": [
         "if (!(Test-Path $persist_dir\\OORegEdt.ini)) {",
-        "   New-Item $dir\\OORegEdt.ini | Out-Null",
+        "   New-Item -ItemType file $dir\\OORegEdt.ini | Out-Null",
```

Positional parameters should go first.
```suggestion
        "   New-Item \"$dir\\OORegEdt.ini\" -ItemType File  | Out-Null",
```